libraroo-backend/src/catalog-books/books-api/books-api.service.ts

```diff
@@ -0,0 +1,28 @@
+import { BadRequestException, Injectable } from '@nestjs/common';
+import { HttpService } from '@nestjs/axios';
+import { Books } from './types/books-api.types';
+import { firstValueFrom } from 'rxjs';
+
+@Injectable()
+export class BooksApiService {
+  key: string;
+  baseUrl: string;
+  constructor(private readonly httpService: HttpService) {
+    this.key = process.env.GOOGLE_API_KEY;
+  }
+  async findByTitleAuthorAndLanguage(
+    author: string,
+    title: String,
+    langCode: string,
+  ): Promise<Books> {
+    const route = `?q=inauthor:"${author}"+intitle:"${title}"&langRestrict=${langCode}&key=${this.key}`;
```

Comment:
Change to `/books/v1/volumes?q=inauthor:....` (remove the added part from the env var, we might want to use other google APIs such as google maps to validate addresses and calculate distances for Borrows)

Reply:
Also, do we want to handle pagination? Currently you will only validate the first 10 books.
FYI This could also be improved by adding a retry mechanism if we receive a 429 "too many requests" status code, not really sure its worth your time